navigation/index.tsx

```diff
@@ -64,32 +64,18 @@ function BottomTabNavigator() {
       }}>
       <BottomTab.Screen
         name="TabOne"
```

Comment:
Nit: Shouldn't the names also be changed to "Profile" and "Settings" to match the new screens?